src/api/controllers/lineEvents.js

```diff
@@ -0,0 +1,89 @@
+
+import mongoose from 'mongoose'
+const User = mongoose.model('User');
+const Line = mongoose.model('Line');
+const LineEvent = mongoose.model('LineEvent');
+
+import response from './response.js';
+
+
+// get /events
+// query: user_id, employer_id, created_by, status
+// future params: limit, sort
+// by default sorted created_by desc
+// ADMIN ONLY
+function getLineEvents(req, res) {
+    console.log(req.user);
+
+    if (!req.user._id) {
+        res.status(401).json({
+            "message": response.unauthorized
+        });
+    // } else if (req.user.user_type !== 'admin') {
```

Comment:
I'll make an issue for this commented out part since we haven't implemented admin user type yet

Reply:
Actually wait, now that I've gone through all the files you did actually add admin user type. So I'm curious why this is still commented out